.gitmodules

```diff
@@ -1,18 +1,21 @@
 [submodule "src/coreclr"]
 	path = src/coreclr
 	url = https://github.com/dotnet/coreclr
+	branch = dev/release/2.0.0
 [submodule "src/roslyn"]
 	path = src/roslyn
 	url = https://github.com/dotnet/roslyn
 [submodule "src/core-setup"]
 	path = src/core-setup
 	url = https://github.com/dotnet/core-setup
+	branch = dev/release/2.0.0
 [submodule "src/standard"]
 	path = src/standard
 	url = https://github.com/dotnet/standard
+	branch = dev/release/2.0.0
 [submodule "src/cli"]
 	path = src/cli
-	url = https://www.github.com/dotnet/cli
+	url = https://github.com/dotnet/cli
 [submodule "src/nuget-client"]
 	path = src/nuget-client
 	url = https://github.com/dagood/NuGet.Client
```

Comment:
Are we OK with this pointing to your fork?

Reply:
No, this is tracked by https://github.com/dotnet/source-build/issues/107. Not trivial to switch to a new fixed commit: see https://github.com/dotnet/source-build/pull/231 for an attempt.